### ISSUE TITLE
Create setup / install instructions

### ISSUE THREAD

**Issue by lo9key** [OregonStateRocketry/30k2018-CS-Capstone#15]
We should include instructions to set up and install our project from scratch.  Ideally we should make this easy with a program that installs the necessary libraries and moves files into the correct locations.

- [x] Install instructions for Avionics
- [x] Install program for Avionics
- [ ] Install instructions for Database
- [x] Install program for Database
- [ ] Install instructions for Display
- [ ] Install program for Display

**Comment by lo9key** [comment]
Avionics steps (in progress):
```
sudo apt-get install mariadb-server python3-pip pigpio
sudo systemctl enable pigpiod // check this
sudo systemctl start pigpiod // check this
pip3 install pyyaml pymysql
```

**Comment by lo9key** [comment]
Ben suggests using a .sh file for the installer.

**Comment by lo9key** [comment]
Addressed by ef542e043e4958a48789cc80b25953c18c719ad4
